src/test/java/org/opensearch/sdk/TestSDKClient.java

```diff
@@ -63,6 +66,15 @@ public void testCreateJavaClient() throws Exception {
         sdkClient.doCloseJavaClient();
     }
 
+    @Test
+    public void testCreateJavaAsyncClient() throws Exception {
+        OpenSearchAsyncClient javaAsyncClient = sdkClient.initializeJavaAsyncClient(settings);
+        assertInstanceOf(OpenSearchIndicesAsyncClient.class, javaAsyncClient.indices());
+        assertInstanceOf(OpenSearchClusterAsyncClient.class, javaAsyncClient.cluster());
+
+        sdkClient.doCloseJavaClient();
```

Comment:
Shouldn't this be `doCloseJavaAsyncClient()` (which doesn't yet exist but should, along with adding to `close()`?)

Reply:
This is how `doCloseJavaClient` looks like:

```java
public void doCloseJavaClient() throws IOException {
        if (restClient != null) {
            restClient.close();
        }
    }
```

We are actually closing the restClient and it's the same for `JavaClient` and `JavaAsyncClient` as I have created a new method to `initalizeTransport` where we actually build the `restClient`. I'm not sure how different `doCloseJavaAsyncClient` gonna look like.